pkgs/by-name/mu/music-assistant/providers.nix

```diff
@@ -150,6 +150,6 @@
         duration-parser
         yt-dlp
         ytmusicapi
-      ];
+      ]; # missing deno
```

Comment:
:rotating_light: :rotating_light: :rotating_light: 

alerta, alerta, fck yt.

Reply:
Isn't this https://pypi.org/project/deno/ ? 😅